check_gha_pinning/__init__.py

```diff
@@ -36,13 +45,24 @@ def _build_github_url(action: str) -> str:
     return f"https://github.com/{repo.group()}"
 
 
-def _get_action_tags(action: str) -> tuple[dict[str, str], dict[str, list[str]]]:
+ActionTagsByTag = dict[str, str]
```

Comment:
```suggestion
ActionShaByTag = dict[str, str]
```

Reply:
See, I contemplated that, but it's really a dict of the action's tags, keyed on the tag. There will be no untagged shas in the list. Thoughts?

Reply:
mhh I see, I guess I was thinking ls-remote gives more than that, but with --tags it really only lists the tags